code/datums/elements/item_scaling.dm

```diff
@@ -0,0 +1,45 @@
+///Bespoke element for scaling items in hand/inventory and in the overworld (on turfs).
+/datum/element/item_scaling
+	element_flags = ELEMENT_BESPOKE
+	id_arg_index = 2
+	var/overworld_scaling
+	var/storage_scaling
+
+///Adds the scaling values set in AddElement to the bespoke element and registers relevant signals.
+/datum/element/item_scaling/Attach(datum/target, overworld_scaling, storage_scaling)
+	. = ..()
+	if(!isatom(target))
+		return ELEMENT_INCOMPATIBLE
+	///Initial scaling set to overworld_scaling when item is spawned.
+	scale(target, overworld_scaling)
+
+	src.overworld_scaling = overworld_scaling
+	src.storage_scaling = storage_scaling
+
+	///Object scaled when placed in an equipment slot OR when entering a storage component.
+	RegisterSignal(target, list(COMSIG_ITEM_EQUIPPED, COMSIG_STORAGE_ENTERED), .proc/scale_storage)
+	///Object scaled when dropped/thrown OR when exiting a storage component.
+	RegisterSignal(target, list(COMSIG_ITEM_DROPPED, COMSIG_STORAGE_EXITED), .proc/scale_overworld)
+
+/datum/element/item_scaling/Detach(datum/target)
+	UnregisterSignal(target, list(COMSIG_ITEM_PICKUP, COMSIG_ITEM_DROPPED, COMSIG_STORAGE_ENTERED,\
+	COMSIG_STORAGE_EXITED))
```

Comment:
```suggestion
	UnregisterSignal(target, list(
		COMSIG_ITEM_PICKUP,
		 COMSIG_ITEM_DROPPED,
		 COMSIG_STORAGE_ENTERED,
		COMSIG_STORAGE_EXITED,
	))
```
This works fine, no need to escape the new line.
Trailing commas are good, as well.